.github/scripts/check-asset-urls.sh

```
@@ -0,0 +1,120 @@
+# #107 reached us, as a bug report. See issue #115.
+#
+# URLs are scraped from the sources rather than listed here, so the check
+# cannot drift away from what the code actually requests.
+#
+# Usage: .github/scripts/check-asset-urls.sh [--markdown]
+# Exit:  0 all reachable, 1 at least one failed.
+
+set -uo pipefail
+
+repo_root="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+cd "$repo_root"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)